platform/Utc.roc

```diff
@@ -26,30 +27,30 @@ now =
 nanosPerMilli = 1_000_000
 
 ## Convert Utc timestamp to milliseconds
-toMillisSinceEpoch : Utc -> U128
+toMillisSinceEpoch : Utc -> I128
 toMillisSinceEpoch = \@Utc nanos ->
     nanos // nanosPerMilli
 
 ## Convert milliseconds to Utc timestamp
-fromMillisSinceEpoch : U128 -> Utc
+fromMillisSinceEpoch : I128 -> Utc
 fromMillisSinceEpoch = \millis ->
     @Utc (millis * nanosPerMilli)
 
 ## Convert Utc timestamp to nanoseconds
-toNanosSinceEpoch : Utc -> U128
+toNanosSinceEpoch : Utc -> I128
 toNanosSinceEpoch = \@Utc nanos ->
     nanos
 
 ## Convert nanoseconds to Utc timestamp
-fromNanosSinceEpoch : U128 -> Utc
+fromNanosSinceEpoch : I128 -> Utc
 fromNanosSinceEpoch = @Utc
 
 ## Calculate milliseconds between two Utc timestamps
-deltaAsMillis : Utc, Utc -> U128
+deltaAsMillis : Utc, Utc -> I128
 deltaAsMillis = \@Utc first, @Utc second ->
     (Num.absDiff first second) // nanosPerMilli
 
 ## Calculate nanoseconds between two Utc timestamps
-deltaAsNanos : Utc, Utc -> U128
+deltaAsNanos : Utc, Utc -> I128
```

Comment:
Makes sense. I'll make that adjustment now.

Reply:
One thing to consider is that since the user would be comparing to `Utc` types, which are I128 backed, the absolute difference will never exceed the magnitude of an I128, so there is not really any need for U128, we are simply adding an unnecessary type cast.

Reply:
I have asked for input from the community [on zulip](https://roc.zulipchat.com/#narrow/stream/231634-beginners/topic/Code.20review.3F/near/419008444).

Reply:
Requested changes are in the PR! ✅